index.js

```diff
@@ -44,15 +42,16 @@ export default function (app, router, store, config) {
       const Button = Vue.extend(PaypalButton)
       const btnInstance = (new Button({ parent: app }))
 
-      var container = document.querySelector('#place-order-container')
+      var orderBtn = document.querySelector('.place-order-btn')
```

Comment:
@collymore For this access to class need merge pull on core https://github.com/DivanteLtd/vue-storefront/pull/1658/files